lib/config-generator.js

```diff
@@ -570,17 +570,16 @@ class ConfigGenerator {
         const contentBase = pathUtil.getContentBase(this.webpackConfig);
 
         const devServerOptions = {
-            contentBase: contentBase,
-            // this doesn't appear to be necessary, but here in case
-            publicPath: this.webpackConfig.getRealPublicPath(),
+            static: {
+                directory: contentBase,
+                // this doesn't appear to be necessary, but here in case
+                publicPath: this.webpackConfig.getRealPublicPath(),
+            },
             // avoid CORS concerns trying to load things like fonts from the dev server
             headers: { 'Access-Control-Allow-Origin': '*' },
-            hot: this.webpackConfig.useHotModuleReplacementPlugin(),
             // required by FriendlyErrorsWebpackPlugin
-            quiet: true,
             compress: true,
             historyApiFallback: true,
-            watchOptions: this.buildWatchOptionsConfig(),
```

Comment:
This now lives under `static.watch`. However, it automatically reads the root-level `watchOptions` config and uses that as its defaults - https://github.com/webpack/webpack-dev-server/blob/788f2ba7d9152610ba0d3024f0079c98aad86991/lib/utils/normalizeOptions.js#L9-L23 - so we don't need to re-set the options here also.